app/src/account.c

```diff
@@ -223,6 +224,26 @@ zxerr_t slot_parseSlot(uint8_t *buffer, uint16_t bufferLen) {
         return zxerr_out_of_bounds;
     }
 
+    if (!path_is_empty(tmp_slot.path)) {
+        uint64_t account = 0;
+        //LE <-> BE conversion necessary to validate account
+        for(int i=0; i<SLOT_ACCOUNT_SIZE; i++) account = 256*account + tmp_slot.account.data[i];
```

Comment:
Good catch to reverse endianness here. 
I suggest to use bit wise operations rather than arithmetic.
```suggestion
        for(int i=0; i<SLOT_ACCOUNT_SIZE; i++) account = (account << 8) | tmp_slot.account.data[i];
```